packages/sync-actions/src/utils/common-actions.js

```diff
@@ -2,6 +2,12 @@ import isNil from 'lodash.isnil'
 import clone from './clone'
 import * as diffpatcher from './diffpatcher'
 
+export function getIsAnEmptyValue(value, shouldOmitEmptyString) {
```

Comment:
Not fixated on this, but it may be a good idea to pass a default value

Reply:
This `getIsBoolean` naming convention is driving me nuts :)